src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -92,6 +92,8 @@ function MoneyRequestView({report, parentReport, shouldShowHorizontalRule, trans
 
     const isDistanceRequest = TransactionUtils.isDistanceRequest(transaction);
 
+    const isInteractive = canEdit && !isSettled;
```

Comment:
This condition is redundant I think. If canEdit, !isSettled already

https://github.com/Expensify/App/blob/643e2e8db576978b87b0acd08cf05acef623b1b0/src/libs/ReportUtils.js#L1264-L1267